src/seam-connect/client.ts

```diff
@@ -111,4 +118,51 @@ export class Seam extends Routes {
       throw error
     }
   }
+
+  static async getClientAccessToken(
+    ops: catParams
+  ): Promise<APIResponse<ClientSessionResponseInterface>> {
+    let params: any = {}
+
+    if (ops.apiKey?.startsWith("seam_test")) {
+      // backend mode
+      params["api_key"] = ops.apiKey
+    } else if (ops.pubKey?.startsWith("seam_pk")) {
+      // frontend mode
+      params["pub_key"] = ops.pubKey
+    }
+    params["user_identifier_key"] = ops.userIdentifierKey
+    try {
+      const response = await axios.post(
+        ops.endpoint + "internal/client_access_tokens/create",
+        params
+      )
+      if (response.data.error) {
+        return {
+          ok: false,
+          error: {
+            type: response.data.error.type,
+            message: response.data.error.message,
+          },
+        }
+      }
+      return await response.data
+    } catch (e: any) {
+      return {
+        ok: false,
+        error: {
+          type: "unknown",
+          message: e.toString() ?? "Something went wrong",
+        },
+      }
+    }
+  }
+}
+
+type catParams = {
```

Comment:
PascalCase types, need linter here

```suggestion
type CATParams = {
```